app/src/test/java/org/oppia/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -902,6 +968,21 @@ class StateFragmentLocalTest {
     testCoroutineDispatchers.advanceTimeBy(TimeUnit.SECONDS.toMillis(10))
   }
 
+  private fun submitTwoWrongAnswersToQuestionPlayer() {
+    submitTwoWrongAnswerToQuestionPlayerFractionInput()
+    submitTwoWrongAnswerToQuestionPlayerFractionInput()
+  }
+
+  private fun submitTwoWrongAnswerToQuestionPlayerFractionInput() {
```

Comment:
Isn't this submitting just 1 wrong answer?

Reply:
Yes, I should update the name of this function. 
Created `submitTwoWrongAnswersToQuestionPlayer()` already which calling this method twice.